src/Tasks/Microsoft.NET.Build.Tasks/JoinItems.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using System.Linq;
+
+namespace Microsoft.NET.Build.Tasks
+{
+    public class JoinItems : TaskBase
```

Comment:
@rainersigwald I don't understand how that batching is working.  The condition seems like it would do the opposite of a join (ie return all pairs where the identities didn't match): `Condition="'%(Identity)' != '@(R) '"`.